modoboa_amavis/sql_connector.py

```diff
@@ -112,25 +107,29 @@ def _get_quarantine_content(self):
                     nfilter = Q(mail__from_addr__contains=pattern)
                 elif crit == "subject":
                     nfilter = Q(mail__subject__contains=pattern)
+                elif crit == "to":
+                    if "str_email" not in self._annotations:
+                        self._annotations["str_email"] = ConvertFrom("rid__email")
+                    nfilter = Q(str_email__contains=pattern)
                 else:
-                    nfilter = self._apply_extra_search_filter(crit, pattern)
-                    if nfilter is None:
-                        continue
+                    continue
                 search_flt = (
                     nfilter if search_flt is None else search_flt | nfilter
                 )
             if search_flt:
                 flt &= search_flt
-        msgtype = self.navparams.get('msgtype', None)
+        msgtype = self.navparams.get("msgtype", None)
         if msgtype is not None:
             flt &= Q(content=msgtype)
 
         flt &= Q(
             mail__in=Quarantine.objects.filter(chunk_ind=1).values("mail_id")
         )
-        messages = Msgrcpt.objects.select_related("mail", "rid").filter(flt)
-        messages = self._apply_extra_select_filters(messages)
-        return messages
+
+        return Msgrcpt.objects\
+            .annotate(**self._annotations)\
+            .select_related("mail", "rid")\
+            .filter(flt)
```

Comment:
I know it is not true everywhere but I now try to avoid adding multilines separated with \, I use () instead. Something like this:
```python
return (
    Msgrcpt.objects
    .annotate(**self._annotations)
    .select_related("mail", "rid")
    .filter(flt)
)
```
What do you think about it?

Reply:
fixed